src/corehost/cli/ijwhost/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required (VERSION 2.6)
+project(ijwhost)
+
+set(DOTNET_PROJECT_NAME "ijwhost")
+
+# Include directories
+include_directories(../fxr)
+include_directories(${ARCH_SPECIFIC_FOLDER_NAME})
+
+# CMake does not recommend using globbing since it messes with the freshness checks
+set(SOURCES
+    ijwthunk.cpp
+    ijwhost.cpp
+    ../fxr_resolver.cpp
+    ../../common/trace.cpp
+    ../../common/utils.cpp
+    ../fxr/fx_ver.cpp
+    pedecoder.cpp
+    bootstrap_thunk_chunk.cpp
+    ${ARCH_SPECIFIC_FOLDER_NAME}/bootstrap_thunk.cpp
+    Exports.def
+)
+
+if(WIN32)
+    list(APPEND SOURCES
+        Exports.def)
```

Comment:
Can we use `__declspec(dllexport)` instead of the DEF file?

Reply:
We can't use `__declspec(dllexport)` because the MSVC compiler (for `_CorDllMain`) and CoreCLR (for `GetTokenForVTableEntry`) are relying on the entry-point being the same name on all platforms instead of accounting for the stdcall mangling. We can make an active decision to change it, but at least for now I've decided to keep it simpler by just using a DEF file to expose an unmangled name on all platforms.